Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -0,0 +1,176 @@
+. ./SonarQubeHelper.ps1
+
+$DefaultNumRetriesForAnalysisToComplete = 120; # there 1s delay between each retry plus the time it takes to make a web call
```

Comment:
It's more usual to express a timeout as a period rather than a number of retries with a fixed interval between them.
It's also more predictable (what's the maximum time it could take the web call to time out?)